packages/block-library/src/post-author/edit.js

```diff
@@ -30,6 +31,14 @@ const AUTHORS_QUERY = {
 	per_page: 100,
 };
 
+const DEFAULT_AVATAR_SIZES = [ 24, 48, 96 ].map( ( size ) => ( {
+	value: size,
+	label: `${ size } x ${ size }`,
+} ) );
+
+const DEFAULT_AVATAR_URL =
+	'https://www.gravatar.com/avatar/00000000000000000000000000000000?d=mp&f=y';
```

Comment:
Is this only used as a fallback?

Reply:
For example, suppose you inserted this block in Single template. On the front end, the author image would be displayed based on the actual articles. However, in Single template, the actual author image URL cannot be retrieved because the post context doesn't exist.

In such cases, avatars should be displayed as placeholders.

Reply:
But this remote url will not always be available, for example on closed networks.
I wonder if there were any prior discussions about it during development of the other avatar blocks.

Reply:
I examined the other blocks and found that the `avatarUrl` was being retrieved by [__experimentalDiscussionSettings()](https://github.com/WordPress/gutenberg/blob/c302e250fdbab8952a1bdf136c48c56c93b04267/packages/block-library/src/comment-author-avatar/edit.js#L43-L47), so I refactored it in 4b5988e. This URL will be `http://1.gravatar.com/avatar/?s=96&d=mm&f=y&r=g`, etc.

I have looked at [#38072](https://github.com/WordPress/gutenberg/pull/38072#discussion_r802927819) and there does not appear to be any discussion on closed networks.

Reply:
Thank you for tracking this down 👍